docs/README.md

```diff
@@ -0,0 +1,272 @@
+# Microsoft Azure SDK for Ruby
+
+As of (2022/01), Azure SDK for Ruby - Resource Management Libraries are officially deprecated. The libraries will be in maintenance mode for a period of 3 years and security / bug fixes will still be applied. However, there will be no further releases based on feature requests. 
+
+We recommend you use Azure REST API directly in Ruby (instructions are provided in this doc)
```

Comment:
"In the future, we recommend that you interface with the Azure REST APIs directly from Ruby; instructions are provided in this doc."